tests/test_utils.py

```diff
@@ -346,28 +346,32 @@ class PlainRelatedResourceTypeSerializer(serializers.Serializer):
 
 
 def test_get_included_serializers():
-    class IncludedSerializersModel(DJAModel):
+    class DeprecatedIncludedSerializersModel(DJAModel):
```

Comment:
I do not see that renaming of the classes are needed here because they are inline. I prefer to keep the test as is so to make sure there are no regressions. `deprecated_call` call needs to be added of course.

Reply:
I remember Django was telling me that it's not possible to do this, perhaps because I moved it outside of the function.
Will try again.

Reply:
Yup, it didn't work.

Reply:
I see there is a conflict between model names... Anyway this code only lives till the next major version so let's leave it like that.